taskcluster/sql/machtry_quantiles.sql

```diff
@@ -0,0 +1,18 @@
+WITH
+  a AS (
+  SELECT
+    APPROX_QUANTILES(total_seconds, 100 IGNORE NULLS) AS quants
```

Comment:
Do we want to take different runs into account above P59 is somewhat bogus as they are greater than a day.  For now merging as is since this is capturing what is already running.